packages/kolibri-zip/src/fileUtils.js

```diff
@@ -68,117 +81,109 @@ const domParser = new DOMParser();
 
 const domSerializer = new XMLSerializer();
 
-const attributes = ['src', 'href'];
-
-const attributesSelector = attributes.map(attr => `[${attr}]`).join(', ');
-
-const urlStyleAttributeSelector = '[style*="url("]';
-
-const audioClassAttributeSelector = '[class*="audio-sentence"], [data-backgroundaudio]';
+const urlAttributes = ['src', 'href'];
 
 const queryParamRegex = /([^?)]+)?(\?.*)/g;
 
 export function getDOMPaths(fileContents, mimeType) {
   const dom = domParser.parseFromString(fileContents.trim(), mimeType);
-  const elements = dom.querySelectorAll(attributesSelector);
-  return flatten(
-    Array.from(elements).map(element =>
-      attributes
-        .map(a => element.getAttribute(a))
-        .filter(Boolean)
-        .map(url => url.replace(queryParamRegex, '$1')),
-    ),
+  // Get paths from URL attributes (src, href)
+  const urlPaths = flatten(
+    urlAttributes.flatMap(attr => {
```

Comment:
This is an error, this should be `map` not `flatMap` as `flatMap` is only supported from iOS Safari 12 and onwards: https://caniuse.com/?search=flatMap